### DISCUSSION TITLE
Table inheritance

### DISCUSSION THREAD

**Discussion by MMartin09** [piccolo-orm/piccolo#894] [Q&A]
Hi, 
is it possibel to implement a "Joined Table Inheritance" similar to SQLAlchemy? https://docs.sqlalchemy.org/en/20/orm/inheritance.html#joined-table-inheritance

**Comment by dantownsend** [comment]
I'm not too familiar with that feature from SQLAlchemy. What use case do you have for it?

You can use mixins if several columns need to share the same columns:

https://piccolo-orm.readthedocs.io/en/latest/piccolo/schema/advanced.html#mixins
